crates/net/network/src/transactions/mod.rs

```diff
@@ -1199,7 +1204,7 @@ impl FullTransactionsBuilder {
     /// Append a transaction to the list if it doesn't exceed the maximum target size.
     fn push(&mut self, transaction: &PropagateTransaction) {
         let new_size = self.total_size + transaction.size;
-        if new_size > POOLED_TRANSACTIONS_RESPONSE_SOFT_LIMIT_BYTE_SIZE {
+        if new_size > FULL_TRANSACTIONS_BROADCAST_MSG_BYTE_SIZE_SOFT_LIMIT {
             return
```

Comment:
this must now ensure that we always have at least 1 tx

Reply:
I reverted this to how it was before https://github.com/paradigmxyz/reth/pull/6301

Reply:
this needs must be changed now that we have a soft limit that is lower than certain transactions